tests/dummy/app/routes/nodes/detail.js

```diff
@@ -21,7 +22,141 @@ export default Ember.Route.extend({
             } else {
                 console.log('You do not have permissions to edit this node');
             }
+        },
+        addContributor(contribId, permission, bibliographic) {
+            var node = this.modelFor(this.routeName);
+            if (contribId) {
+                if (node.get('currentUserPermissions').indexOf('admin') !== -1) {
```

Comment:
Adding a TODO for myself to update permissions strings when https://github.com/CenterForOpenScience/ember-osf/pull/23/files#diff-7fd0bf247bef3c257e0fcfd7e544a338R5 is merged.